From 1a85228cf1727a2fa9719980ec79084a878748f6 Mon Sep 17 00:00:00 2001
From: Romain Rigaux <romain@cloudera.com>
Date: Wed, 25 Feb 2015 18:14:30 -0800
Subject: [PATCH 0926/1173] HUE-2102 [oozie] Workflow with credentials can't
 be used with Coordinator

---
 desktop/libs/liboozie/src/liboozie/submission2.py |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/desktop/libs/liboozie/src/liboozie/submission2.py b/desktop/libs/liboozie/src/liboozie/submission2.py
index 4accc54..47fd627 100644
--- a/desktop/libs/liboozie/src/liboozie/submission2.py
+++ b/desktop/libs/liboozie/src/liboozie/submission2.py
@@ -141,6 +141,10 @@ class Submission(object):
       LOG.exception(msg)
       raise PopupException(message=msg, detail=str(ex))
 
+    if self.api.security_enabled:
+      jt_address = cluster.get_cluster_addr_for_job_submission()
+      self._update_properties(jt_address) # Needed for coordinator deploying workflows with credentials
+
     oozie_xml = self.job.to_xml(self.properties)
     self._do_as(self.user.username , self._copy_files, deployment_dir, oozie_xml)
 
-- 
1.7.9.5

